backend/pkg/msgpack/config.go

```diff
@@ -0,0 +1,37 @@
+package msgpack
+
+import (
+	"fmt"
+)
+
+// Config represents the message pack config.
+type Config struct {
+	Enabled bool `json:"enabled"`
```

Comment:
We use the yaml struct tags everywhere

Reply:
ah this was left out. fixed in the new commit